benchmarking/benchmarking.go

```diff
@@ -59,8 +49,20 @@ func RunDispatchCall(b *testing.B, testFn func(i *Instance), components ...*line
 		}
 	}
 
-	// todo output file path flag (fmt.Fprintf accepts writer as first arg)
-	fmt.Printf("median slope analysis: %#v\n", medianSlopesAnalysis(results))
+	// generate weight file
+	analysis := medianSlopesAnalysis(results)
+	fmt.Println(analysis.String())
+
+	if Config.GenerateWeightFiles {
+		template, err := ExtrinsicWeightTemplate()
+		if err != nil {
+			b.Fatalf("failed to get extrinsic weight template")
```

Comment:
Return `b.Fatalf("failed to get extrinsic weight template %v", err.Error())` to report the actual reason behind the error, since parsing is done as part of `ExtrinsicWeightTemplate()`.

Additionally, it would be better to rename it to something that gives a hint of its functionality: `InitExtrinsicWeightTemplate()`, `ProcessExtrinsicWeightTemplate()`, ...